DeskReserve_Tests/RoomControllerTest.cs

```diff
@@ -0,0 +1,155 @@
+﻿using DeskReserve.Domain;
```

Comment:
Separate the tests in folder structure similar in the src, Also fix the folder structure in the src